snellejelle99/DARTS
# #27 Main menu voor applicatie aangemaakt

### Omschrijving
Main menu met drie knoppen aangemaakt. Knoppen navigeren (na het integreren van de die schermen) naar het start match scherm en de speler en match overview schermen. 

**Risico's**
Laag risico: nieuwe UI

---
### Tests

**Test 1 "Hoofdscherm opstarten" :**

Voorbereiding:
- Start het programma op.

Test
- Check of het juiste scherm wordt weergegeven.

Geslaagd als :
Het programma is opgestart als het hoofdscherm met de drie knoppen: 
- [x] Knop naar "Start match" scherm
- [x] Knop naar "Match overzicht" scherm
- [x] Knop naar "Speler overzicht" scherm
---

closes #27
